.slugingore

```diff
@@ -0,0 +1,7 @@
+.flaskenv
+.git
+.github
+.vscode
+.pullpreview
+.pre-commit-config.yaml
+tests
```

Comment:
Maybe also, just for completeness/extra safety:
- `.mypy_cache`
- `.ruff_cache`
- `.import_linter_cache`
- `.idea`
- `.venv`?
- `.pytest_cache`